okhttp/third_party/okhttp/java/io/grpc/okhttp/internal/Platform.java

```diff
@@ -105,47 +119,71 @@ public void connectSocket(Socket socket, InetSocketAddress address,
   }
 
   /** Attempt to match the host runtime to a capable Platform implementation. */
+  @SuppressWarnings("unchecked")
```

Comment:
This is a scary number of changes to this file, such that it would be prohibitive when upgrading. It seems like the delta could be reduced.